impl/connections/s3/src/main/resources/i18n/pentaho-s3/messages_en_US.properties

```diff
@@ -22,3 +22,5 @@ S3.Label.Browse=Browse...
 S3.Label.No=No
 S3.Label.Default=Default
 S3.Label.NA=N/A
+S3.Label.YES=Yes
```

Comment:
@rmansoor 2 things here...
1. the key for Yes/No is different than the messages.properties file (Yes vs YES and No vs NO).
2. Shouldn't this messages file be identical to the messages.properties file? i.e. also include Endpoint, PathStyleAccess, and SignatureVersion as well as have same case of the yes/no keys?